docs/cloud/netdata-functions.md

```diff
@@ -33,7 +33,8 @@ functions - [plugins.d](https://github.com/netdata/netdata/blob/master/collector
 | Function | Description | plugin - module |
 | :-- | :-- | :-- |
 | processes | Detailed information on the currently running processes on the node. | [apps.plugin](https://github.com/netdata/netdata/blob/master/collectors/apps.plugin/README.md) |
-| ebpf_thread | Controller for eBPF threads. | [ebpf.plugin](https://github.com/netdata/netdata/blob/master/collectors/ebpf.plugin/README.md) |
+| ebpf_socket | Detailed socket information. | [ebpf.plugin](https://github.com/netdata/netdata/blob/master/collectors/ebpf.plugin/README.md#function) |
```

Comment:
There is a section `ebpf_socket` in README, maybe to point there?

Reply:
Good suggestion, I used the github visualization to figure out the possible links and I changed it. Thanks!